src/styles/_catalog.scss

```diff
@@ -14,9 +14,22 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+
+    transition: $translation-time ease-in-out;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  &__card:hover {
+    transform: scale(120%, 120%) ;
+    color: #34568b;
```

Comment:
Use nesting. More details in [video](https://mate.academy/learn/css-advanced/sass#/video/925)